client/i18n/locales/english/intro.json

```
@@ -1734,7 +1734,7 @@
       },
       "lecture-working-with-links": {
         "title": "Working with Links",
-        "intro": ["Learn about using links in HTML with these lecture videos."]
+        "intro": ["In these lecture videos, you will learn about the different <code>target</code> attribute values, absolute and relative links and the different links states."]
```

```suggestion
        "intro": [
          "In these lecture videos, you will learn about the different <code>target</code> attribute values, absolute and relative links and the different links states."
        ]
```
To make prettier happy.